apis/pkg/v1/interfaces.go

```diff
@@ -114,6 +132,12 @@ type Package interface { //nolint:interfacebloat // TODO(negz): Could we break t
 
 	GetCommonLabels() map[string]string
 	SetCommonLabels(l map[string]string)
+
+	GetAppliedImageConfigRefs() []ImageConfigRef
+	SetAppliedImageConfigRefs(refs ...ImageConfigRef)
+
+	GetStatusPackage() string
+	SetStatusPackage(s string)
```

Comment:
Open to suggestions on naming for these. They're the status equivalent of `GetSource`/`SetSource`.

Reply:
Updated to `ResolvedSource`.